.github/workflows/ci.yml

```diff
@@ -82,6 +82,12 @@ jobs:
         repository: sbt/zinc
         ref: 1.10.x
         path: zinc
+    - uses: graalvm/setup-graalvm@v1
+      with:
+        java-version: '23'
+        native-image-musl: 'true'
+        set-java-home: 'false'
+        github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
As as FYI, we produce sbtn in a different repo, so we'd likely need to copy-paste this setting and implement a manual version for QEMU ARM Linux emulation - https://github.com/sbt/sbtn-dist, which I was doing last night coincidentally - https://github.com/eed3si9n/sbtn-dist/commit/e166373ae925503cad6a2b7e9e58aaca02bba150

Reply:
If we don't make to make a new release soon we can wait for Github Action arm64 runner which are planned to [arrive by EOY](https://github.com/orgs/community/discussions/19197#discussioncomment-10550689).

Reply:
So it turns out that `--libc=musl` is not supported on ARM Linux:
https://www.graalvm.org/21.3/reference-manual/native-image/ARM64/index.html